client/src/types/type.d.ts

```diff
@@ -9,4 +9,29 @@ declare module 'GlobalType' {
     year: number;
     month: number;
   }
+
+  interface LabelData {
+    idx: number;
+    title: string;
+    color: string;
+    amount: number;
+    unit: string;
+  }
+
+  export interface Tasks {
```

Comment:
의미상 `Task`가 맞을 것 같습니다
```suggestion
  export interface Task {
```